qtconsole/rich_jupyter_widget.py

```diff
@@ -212,19 +213,30 @@ def _is_latex_math(self, latex):
                 return True
         return False
 
+    def _get_colors(self, color):
+        """Get colors from the current syntax style if loadable"""
+        try:
+            return get_colors(self.syntax_style)[color]
+        except ClassNotFound:
+            # the syntax_style has been sideloaded (e.g. by spyder)
+            # in this case the overloading class should override this method
+            return get_colors('default')[color]
+
     def _append_latex(self, latex, before_prompt=False, metadata=None):
         """ Append latex data to the widget."""
         png = None
 
         if self._is_latex_math(latex):
-            png = latex_to_png(latex, wrap=False, backend='dvipng')
+            png = latex_to_png(latex, wrap=False, backend='dvipng',
+                               color=self._get_colors('fgcolor'))
```

Comment:
This assumes that IPython 7.8.0+ is installed, but that couldn't be the case since Qtconsole doesn't have a direct requirement on IPython.

Please handle that by leaving the previous code if IPython's version is less than 7.8.0, and the one you added otherwise. You can use `IPython.version_info` for that comparison.

Reply:
Sorry, please ignore my comment above. We can deal more simply with this issue by requiring Python 3.7+ in our setup.py, which will pull IPython 7.17, according to its docs:

https://ipython.readthedocs.io/en/stable/whatsnew/version7.html#ipython-7-17